docs/usage/configuration/config_documentation.md

```diff
@@ -230,6 +230,9 @@ Example configuration:
 presence:
   enabled: false
 ```
+
+The `enabled_for_sync` sub-option can be used to selectively enable/disable
```

Comment:
Presumably this is an optional boolean---can we say as much, and describe the default value?

Reply:
Err, also: where does presence information show up if not in sync? E.g. Can clients query a given user's presence state directly?

Reply:
> Can clients query a given user's presence state directly?

Yes, there's a GET API: https://spec.matrix.org/v1.8/client-server-api/#get_matrixclientv3presenceuseridstatus

Reply:
I think this is probably misnamed, it really means if users can set presence or not.